Spec/Doubles/CedarDoubleARCSharedExamples.mm

```diff
@@ -42,8 +43,14 @@ myDouble stub_method("methodWithBlock:").and_do(^(NSInvocation *invocation) {
                     [myDouble methodWithBlock:^{ }];
                 });
 
+                // just a no-op method so the runloop has some operations to run
+                NSTimer *timer = [NSTimer timerWithTimeInterval:0.1
+                                                         target:myDouble
+                                                       selector:@selector(value)
+                                                       userInfo:nil
+                                                        repeats:NO];
                 while (!called) {
-                    [[NSRunLoop currentRunLoop] addTimer:[NSTimer timerWithTimeInterval:0.1 invocation:nil repeats:NO] forMode:NSDefaultRunLoopMode];
+                    [[NSRunLoop currentRunLoop] addTimer:timer forMode:NSRunLoopCommonModes];
                     NSDate *futureDate = [NSDate dateWithTimeIntervalSinceNow:0.1];
```

Comment:
This code is not necessary, as far as I have been able to tell.  My changes remove it altogether.